juju/juju
# [JUJU-919] Allow download centos7 series

The following also uses the same code path that computes the base
channel for downloading a charm, which is also the same code path for
deploying a charm for a given series.
## QA steps

*Please replace with how we can verify that the change works.*

```sh
$ juju download slurmdbd --series=centos7
```
